modules/terraform/apply.go

```diff
@@ -40,8 +41,28 @@ func Apply(t *testing.T, options *Options) string {
 	return out
 }
 
+// ApplyAllTg runs terragrunt apply with the given options and return stdout/stderr. Note that this method does NOT call destroy and
+// assumes the caller is responsible for cleaning up any resources created by running apply.
+func ApplyAllTg(t *testing.T, options *Options) string {
+	out, err := ApplyAllTgE(t, options)
+	if err != nil {
+		t.Fatal(err)
+	}
+	return out
+}
+
 // ApplyE runs terraform apply with the given options and return stdout/stderr. Note that this method does NOT call destroy and
 // assumes the caller is responsible for cleaning up any resources created by running apply.
 func ApplyE(t *testing.T, options *Options) (string, error) {
 	return RunTerraformCommandE(t, options, FormatArgs(options, "apply", "-input=false", "-lock=false", "-auto-approve")...)
 }
+
+// ApplyAllTgE runs terragrunt apply-all with the given options and return stdout/stderr. Note that this method does NOT call destroy and
+// assumes the caller is responsible for cleaning up any resources created by running apply.
+func ApplyAllTgE(t *testing.T, options *Options) (string, error) {
+	if options.TerraformBinary != "terragrunt" {
+		return "", fmt.Errorf("terragrunt must be set as TerraformBinary to use this method")
```

Comment:
Can you define a new error type for this? I know we don't have this pattern in this module, but using concrete error types helps with type checking to distinguish between this error and an error returned by the command call. You can take a look at [the aws module errors.go](https://github.com/gruntwork-io/terratest/blob/master/modules/aws/errors.go) for an example.